config/routes.rb

```diff
@@ -1639,6 +1639,7 @@
         alert_profile_edit
         alert_profile_field_changed
         button
+        clear_search_text
```

Comment:
@GregP since this method is a generic method, this route should be added for other ocntrollers as well, why only for miq_policy controller. Try moving this route to common routes related to advanced search related routes https://github.com/GregP/manageiq/blob/55f09e87aa9b713cc1681ec8b29966f386dc6e3b/config/routes.rb#L10